### DISCUSSION TITLE
Can I tween any public property of any MonoScripts instead of modify the tween lib?

### DISCUSSION THREAD

**Discussion by jane00** [jeffreylanters/unity-tweens#35] [Q&A] [answered]
Can I tween any public property of any MonoScripts instead of modify the tween lib?

**Comment by jeffreylanters** [comment, marked as answer]
Hi! Yes, this can be done by using the TweenValue methods such as the [TweenValueFloat](https://github.com/jeffreylanters/unity-tweens#tween-value-float) and [TweenValueVector3](https://github.com/jeffreylanters/unity-tweens#tween-value-vector3) methods.

```cs
// Tween a float from its current value to 100.
myMonobehaviour.TweenValueFloat (100, 1, (v) => {
  myMonobehaviour.myParam = v;
}).SetFrom(myMonobehaviour.myParam);
```
